src/core/Akka.Tests/Actor/FSMTimingSpec.cs

```diff
@@ -365,7 +365,7 @@ public StateMachine(IActorRef tester)
                             var contextLocal = Context;
                             var numberOfMessages = ((ActorCell) contextLocal).Mailbox.NumberOfMessages;
                             SetTimer("hallo", new Tock(), TimeSpan.FromMilliseconds(1));
-                            StaticAwaitCond(() => contextLocal.AsInstanceOf<ActorCell>().Mailbox.HasUnscheduledMessages, TimeSpan.FromSeconds(1), TimeSpan.FromMilliseconds(50));
+                            StaticAwaitCond(() => contextLocal.AsInstanceOf<ActorCell>().Mailbox.HasMessages, TimeSpan.FromSeconds(1), TimeSpan.FromMilliseconds(50));
```

Comment:
Name of the property is now `HasMessages`. Scheduling is solely determined by the `MessageDispatcher`.